src/fhir/api.test.ts

```diff
@@ -20,7 +20,7 @@ const checkResponseOK = async (response: Response) => {
     console.error(r.body);
     throw new Error(response.statusText);
   }
-  if (!(r.type === "Response")) {
+  if (!(r.type === "bundle-response")) {
```

Comment:
Might be personal preference but on reading this I missed the `!` at the start and was surprised. To me this makes it more clear but also happy for you to ignore this and merge anyway
```suggestion
  if (r.type !== "bundle-response") {
```